src/main/java/org/simplify4u/plugins/sign/ArtifactSigner.java

```diff
@@ -76,12 +76,13 @@ protected SignResult makeSignature(InputStream inputStream,
             String extension) {
 
         String targetExt = extension + ".asc";
+        String targetName = artifactId + '-' + version;
 
-        String targetName = artifactId;
         if (classifier != null && !classifier.isEmpty()) {
             targetName += "-" + classifier;
         }
-        targetName += '-' + version + '.' + targetExt;
+
+        targetName += "." + targetExt;
```

Comment:
Just for my own couriosity: Why do you make a `String` out of a single `char`? Is this a special performance trick?

Reply:
I have not noticed it ... probably compilator take care about it